benchmarks/drop/drop.py

```diff
@@ -0,0 +1,381 @@
+"""DROP: A Reading Comprehension Benchmark Requiring Discrete Reasoning Over Paragraphs
+
+Dheeru Dua, Yizhong Wang, Pradeep Dasigi,
+Gabriel Stanovsky, Sameer Singh, and Matt Gardner
+https://arxiv.org/pdf/1903.00161
+
+Based on: https://github.com/openai/simple-evals/blob/main/drop_eval.py
+and https://github.com/EleutherAI/lm-evaluation-harness/tree/main/lm_eval/tasks/drop
+
+# run with default fewshot samples (3)
+inspect eval drop/drop.py
+
+# run with more or no fewshot samples
+inspect eval drop/drop.py -T fewshot=5
+inspect eval drop/drop.py -T fewshot=false
+"""
+
+import re
+import string
+from typing import Dict, List, Tuple, Union
+
+import numpy as np
+from scipy.optimize import linear_sum_assignment
+
+from inspect_ai import Task, task
+from inspect_ai.dataset import Sample, hf_dataset
+from inspect_ai.scorer import Score, Target, bootstrap_std, mean, scorer
+from inspect_ai.solver import (
+    Generate,
+    TaskState,
+    generate,
+    prompt_template,
+    solver,
+    system_message,
+)
+
+DROP_PROMPT_TEMPLATE = {
```

Comment:
Would make these separate constants (same request made in the other PR). Less lines of code, more clear delineation of the various prompts.

Reply:
That makes sense, done.